### PR TITLE
Otp graph status and versions

### PR DESCRIPTION
**Pull request by mansoor-sajjad**
OTP graph status og versions are moved to the right side, which can be hidden and shown by clicking on the button on the top right corner.
Shown by default and buttons have tooltips.

**Shown:**
image

**Hidden:**
image

**Tooltip text:**
image

Feedback please. @testower @vpaturet

### CODE REVIEW

File: src/actions/SuppliersActions.js [hunk 1]
```diff
@@ -795,6 +795,28 @@ SuppliersActions.getGraphStatus = () => async (dispatch, getState) => {
...
+  const url = window.config.timetableAdminBaseUrl + `routing_graph/graphs`;
+
+  return axios
+    .get(url, await getApiConfig(getState().UserReducer.auth))
+    .then(response => {
+      const graphVersions = {
+        streetGraphs: [],
+        transitGraphs: []
+      };
+      console.log('getOTPGraphVersions response = ', response);
```
Comment (testower) [inline src/actions/SuppliersActions.js:808]: Do we need to log this to the console?

**Comment by testower** [comment]
> Firebase CLI v12.7.0 is incompatible with Node.js v14.18.3 Please upgrade Node.js to version >=16.13.0 || >=18.0.0

**Comment by vpaturet** [comment]
Looks good, this covers all the functional needs.
A couple of remarks:
- we want to monitor the graph status continuously, while detailed graph information is required only during roll out of a new OTP version. It would be better to always show the graph status and display detailed information only on demand.
- we need to see the graph creation date/time down to the second (possibly by hovering the mouse over the delay since creation or by replacing the delay since creation by the creation date/time)

**Comment by vpaturet** [comment]
Just to clarify the previous comment: if the detailed graph information is always displayed, it reduces the available space for displaying the deployment pipelines.

**Comment by testower** [comment]
@mansoor-sajjad node-sass is deprecated and doesn't work with newer versions of node, so we need to switch that to (dart-)sass

**Comment by mansoor-sajjad** [comment]
> Just to clarify the previous comment: if the detailed graph information is always displayed, it reduces the available space for displaying the deployment pipelines.

@vpaturet We can hide and show the graph information with the new button on the top right corner. The graph information will be shown by default. But if we need more space, we can always hide it.

**Pull request merged**